en/uavcan/node_firmware.md

```diff
@@ -1,128 +1,72 @@
 # UAVCAN Firmware Upgrading
 
-PX4 will automatically upgrade firmware on UAVCAN nodes if the matching firmware is supplied.
+PX4 will automatically upgrade firmware on UAVCAN nodes if the appropriate firmware is supplied.
+The firmware build process is manufacturer dependent (instructions for different firmware linked below).
 
 :::warning
 UAVCAN devices typically ship with appropriate firmware preinstalled. 
 These instructions are primarily needed when developing UAVCAN devices.
 :::
 
-## Vectorcontrol ESC Codebase (Pixhawk ESC 1.6 and S2740VC)
+## Flashing/Updating a UAVCAN Binary
 
-Download the ESC code:
+To flash the UAVCAN device, all you need to do is copy the UAVCAN firmware binary into the root directory of the flight controller’s SD card and reboot.
 
-```sh
-git clone https://github.com/thiemar/vectorcontrol
-cd vectorcontrol
-```
-
-### Flashing the UAVCAN Bootloader
+After restarting the flight controller the UAVCAN device will automatically be updated with new firmware.
+If successful the firmware binary will be removed from the root directory and there will be a file named **XX.bin** in the **/ufw** directory of the SD card.
 
-Before updating firmware via UAVCAN, the Pixhawk ESC 1.6 requires the UAVCAN bootloader be flashed.
-To build the bootloader, run:
+:::note
+UAVCAN firmware is usually created as part of the build process.
+Commonly the firmware image is built in **build/_name_of_your_uavcan_device_**,  where _name_of_your_uavcan_device_ is specific to the device you are flashing (e.g. **build/ark_can-flow_default**), and will be named with the format **XX-X.X.XXXXXXXX.uavcan.bin**.
```

Comment:
Is it actually named `XX-X.X.XXXXXXXX.uavcan.bin` or are the X placeholders for versions or something else? Does it matter as long as the extension is correct?

Reply:
X is placeholding for numbers, but to my knowledge nothing else will have the same '.uavcan.bin' extension either.

Reply:
Thanks. That's OK, just wanted to make sure that "format" was the right word to use. I think people will infer the extension is required.